source/install/install-ubuntu-2004-postgresql.rst

```diff
@@ -0,0 +1,104 @@
+.. _install-ubuntu-1804-postgresql:
+
+Installing PostgreSQL Database Server
+=====================================
+
+Install and set up the database for use by the Mattermost server. You can install either PostgreSQL or MySQL.
+
+Assume that the IP address of this server is 10.10.10.1.
+
+**To install PostgreSQL on Ubuntu Server 20.04:**
+
+1. Log in to the server that will host the database and issue the following command:
+
+  ``sudo apt install postgresql postgresql-contrib``
+
+  When the installation is complete, the PostgreSQL server is running, and a Linux user account called *postgres* has been created.
+
+2. Log in to the *postgres* account.
+
+  ``sudo --login --user postgres``
+
+3. Start the PostgreSQL interactive terminal.
+
+`` sudo -u postgres psql``
+
+4.  Create the Mattermost database.
+
+  ``postgres=# CREATE DATABASE mattermost;``
+
+5.  Create the Mattermost user 'mmuser'.
+
+  ``postgres=# CREATE USER mmuser WITH PASSWORD 'mmuser-password';``
+
+  .. note::
+    Use a password that is more secure than 'mmuser-password'.
+
+6.  Grant the user access to the Mattermost database.
+
+  ``postgres=# GRANT ALL PRIVILEGES ON DATABASE mattermost to mmuser;``
+
+7. Exit the PostgreSQL interactive terminal.
+
+  ``postgres=# \q``
+
+8. Log out of the *postgres* account.
+
+  ``exit``
+
+9. (Optional) If you use a different server for your database and the Mattermost app server, you may allow PostgreSQL to listen on all assigned IP Addresses. To do so, open ``/etc/postgresql/10/main/postgresql.conf`` as root in a text editor. As a best practice, ensure that only the Mattermost server is able to connect to the PostgreSQL port using a firewall.
```

Comment:
```suggestion
9. (Optional) If you use a different server for your database and the Mattermost app server, you may allow PostgreSQL to listen on all assigned IP addresses. To do so, open ``/etc/postgresql/10/main/postgresql.conf`` in a text editor as *root* user. As a best practice, ensure that only the Mattermost server is able to connect to the PostgreSQL port using a firewall.
```

Reply:
I'm 4/5 for removing "app" from "Mattermost app server". @fakela, did you notice that in any of the existing documentation?

Reply:
Same here, lemme make the needed change